<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Page</class>
 <widget class="QWidget" name="Login_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#Login_PB{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#Login_PB:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#Login_PB:hover{
	background-color:rgba(255, 107, 107, 255);
}

QLabel#error_message_Label{
	color:#cd0000;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="SMT_Label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>411</width>
        <height>101</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="pixmap">
       <pixmap>../Login/Trail_04/icons1/aa.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="username_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Admin Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_LineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_LineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Forgot_Password_PB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>610</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Forgot Password ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_message_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>error</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_LineEdit</tabstop>
  <tabstop>password_LineEdit</tabstop>
  <tabstop>Login_PB</tabstop>
  <tabstop>Forgot_Password_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
